internal/ingress/annotations/sessionaffinity/main.go

```diff
@@ -160,6 +164,11 @@ func (a affinity) Parse(ing *networking.Ingress) (interface{}, error) {
 		am = ""
 	}
 
+	cb, err := parser.GetStringAnnotation(annotationAffinityCanaryBehavior, ing)
+	if err != nil {
+		cb = ""
+	}
+
```

Comment:
Suggestion:

As for now we only have sticky and legacy, being sticky the default, why not doing something like:

```suggestion
	cb, err := parser.GetStringAnnotation(annotationAffinityCanaryBehavior, ing)
	if err != nil || cb != "legacy" {
		cb = "sticky"
	}
```
This way we early assume that sticky is the value to be returned here. 

Or even doing like https://github.com/kubernetes/ingress-nginx/blob/cb6ccdc00d2c0ea99b38437fd3c1b60dd34609a3/internal/ingress/annotations/sessionaffinity/main.go#L104, defining a constant of defaultAffinityCanaryBehavior here as "sticky" and returning this as the default